SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -54,6 +58,8 @@ boolean ultrasonic_activation = true;
 boolean servo_activation = true;     
 boolean motor_activation = true;
 boolean all_activation = true; //Used for de-_activation of all sensors/modules 
+boolean camera_servo_activation_horizontal = true; //X is horizontal  
+boolean camera_servo_activation_vertical = true; //Y is vertical
```

Comment:
Should be "camera_servo_horizontal_activation" and "camera_servo_vertical_activation for consistency"